cwms-data-api/src/test/java/cwms/cda/data/dao/TimeSeriesDaoTest.java

```diff
@@ -171,7 +169,7 @@ public void testCreateWithData() throws Exception
 			int count = 60 / 15; // do I need a +1?  ie should this be 12 or 13?
 			// Also, should end be the last point or the next interval?
 
-			TimeSeries ts = new TimeSeries(null, -1, 0, tsId, officeId, start, end, "m", Duration.ofMinutes(minutes));
+			TimeSeriesWithDate ts = new TimeSeriesWithDate(null, -1, 0, tsId, officeId, start, end, "m", Duration.ofMinutes(minutes));
```

Comment:
Shouldn't be able to store time series with a data-entry-date, no need for this test to change.

Reply:
Reverted test, test is disabled due to deprecated connection usage